### DISCUSSION TITLE
Problem while installing

### DISCUSSION THREAD

**Discussion by anishshriram** [mlcommons/GaNDLF#413] [Q&A] [answered]
I am trying to install GaNDLF again following the steps provided on the doc website.

When I write the command ```pip install -e .``` I get the following error:
```
ERROR: Could not find a version that satisfies the requirement torch==1.8.2 (from gandlf) (from versions: 1.0.0, 1.0.1, 1.0.1.post2, 1.1.0, 1.2.0, 1.3.0, 1.3.1, 1.4.0, 1.5.0, 1.5.1, 1.6.0, 1.7.0, 1.7.1, 1.8.0, 1.8.1, 1.9.0, 1.9.1, 1.10.0, 1.10.1, 1.10.2, 1.11.0)
ERROR: No matching distribution found for torch==1.8.2
```
I don't believe there is a version 1.8.2, so I am not sure why that is a requirement. Am I doing something wrong?

If I try to run ```python ./gandlf_verifyInstall``` on the other hand, it seems to have no issues, so maybe the error is not significant?
```
GaNDLF installed version: 0.0.14-dev
GANDLF's git submodules were successfully imported.
GaNDLF is ready. See https://cbica.github.io/GaNDLF/usage
```

For context these were the steps I completed prior:
```
git clone https://github.com/CBICA/GaNDLF.git
cd GaNDLF
conda create -n venv_gandlf_4 python=3.7 -y
source activate venv_gandlf_4
conda install pytorch torchvision torchaudio cudatoolkit=10.2 -c pytorch
conda install -c conda-forge gandlf -y
pip install -e . # This is the line im having trouble with
```

**Comment by sarthakpati** [comment, marked as answer]
Hi,

In https://cbica.github.io/GaNDLF/setup, there is a step for torch installation based on the specific CUDA version:

```bash
### PyTorch LTS installation - https://pytorch.org/get-started/locally
## CUDA 10.2
# conda install pytorch torchvision torchaudio cudatoolkit=10.2 -c pytorch-lts -y
## CUDA 11.1
# conda install pytorch torchvision torchaudio cudatoolkit=11.1 -c pytorch-lts -c conda-forge -y
```

If you have installed torch properly, `pip install -e .` should work.

**Reply by anishshriram** [reply]
Hello,

So sorry for the delayed confirmation, I did not see your response. I started again from scratch, and everything worked smoothly. I appreciate your help!
